src/python/pants/backend/codegen/protobuf/python/python_protobuf_subsystem.py

```diff
@@ -32,11 +34,24 @@ def register_options(cls, register):
                 "be automatically added to every `protobuf_library` target"
             ),
         )
+        register(
+            "--mypy-plugin",
+            type=bool,
+            default=False,
```

Comment:
I decided to model this as a boolean toggle, rather than a string that lets you determine which specific version of `mypy-protobuf` is used, where being unset means to not use the plugin. 

I figure the boolean is much simpler and "just works" out of the box. But users may want more power to change this, as the plugin version does depend on what MyPy version you're using; unclear to me.

Reply:
we are okay with the "just works" approach. It's simpler.

Reply:
See below, i think the version should be a separate option.